packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.tsx

```diff
@@ -48,6 +57,8 @@ export type AccessRequirementListProps = {
   onHide: () => void
   /* Displays the provided list of access requirements, instead of the ones fetched using the subject ID */
   accessRequirementFromProps?: Array<AccessRequirement>
+  /* If provided, displays these actions on when viewing all ARs */
+  customDialogActions?: React.ReactNode
```

Comment:
Instead of showing this component outside of a modal next to custom actions, pass the custom actions into this component